### PR TITLE
Eway Bill for Sales Invoice

### PR DESCRIPTION
**Pull request by finbyz**
Reference:- [#15632](https://github.com/frappe/erpnext/pull/15632)
We have modified our PR as suggested by @sagarvora in for [this](https://github.com/frappe/erpnext/pull/15632#issuecomment-428822018) comment from the above PR.

### CODE REVIEW

File: erpnext/regional/india/setup.py [hunk 1]
```diff
@@ -201,6 +201,35 @@ def make_custom_fields(update=True):
...
+			insert_after='gst_transporter_id', fetch_from='driver.full_name', print_hide=1),
+		dict(fieldname='lr_date', label='Transport Receipt Date', fieldtype='Data', 
+			insert_after='driver_name', default='Today', print_hide=1),
+		dict(fieldname='mode_of_transport', label='Mode of Transport', fieldtype='Select', 
+			insert_after='lr_date', options='\nRoad\nAir\nRail\nShip', default='Road', translatable=1, print_hide=1),
+		dict(fieldname='gst_vehicle_type', label='GST Vehicle Type', fieldtype='Select', 
+			insert_after='mode_of_transport', options='\nRegular\nOver Dimensional Cargo (ODC)', default='Regular',
+			depends_on='eval:(doc.mode_of_transport === "Road")', translatable=1, print_hide=1),
+	]
+
```
Comment (finbyz) [inline erpnext/regional/india/setup.py]: Added Transporter Info section in Sales Invoice

**Comment by rmehta** [comment]
Can you please add a screenshot of the e-way bill? And maybe add simple test to check the output of the bill?

**Comment by sagarvora** [comment]
@finbyz Thanks for making the necessary changes.

I've planned a feature development to generate a JSON file from DN/SINV itself based on the spec provided on eWaybill website, but its not on priority right now.

Can you please add a screenshot for the Transporter Info section in Sales Invoice (after patch is applied)?

Also a good idea might be ensuring the patch (https://github.com/frappe/erpnext/blob/staging-fixes/erpnext/patches/v11_0/ewaybill_fields_gst_india.py)  runs for those already on v11, but I don't know how that can be implemented.

**Comment by finbyz** [comment]
@sagarvora Here is the screenshot of Transporter Info Section after patch is applied.

screenshot 6

And secondly, we have already applied the patch on ([https://github.com/frappe/erpnext/blob/staging-fixes/erpnext/patches/v11_0/ewaybill_fields_gst_india.py](https://github.com/frappe/erpnext/blob/staging-fixes/erpnext/patches/v11_0/ewaybill_fields_gst_india.py)) that you provided.

**Comment by test2a** [comment]
@finbyz any idea if and when the latest update on eway bill would be implemented?

**Pull request closed without merging**